README.md

```diff
@@ -1,8 +1,24 @@
 ![alt tag](https://66.media.tumblr.com/tumblr_lfp90xpDTm1qb9w8so1_250.gif)
 
+# Ping Pong Championship Rules
 
-- New rules must be voted on and have a 2/3 majority in favor of the new rule.
-- If the existing champion is out of the office for 3 days, regardless of reason, excluding weekends, then they give up their title.
-- If the champion challenges someone or accepts a challenge and do not play then they forfiet their title to the opponent.
-- The winner must win 3/5 games.
-- If someone wins 3 consecutive games then they must offer a winner takes all match. The winner of the winner takes all match wins the title.
+## Article 1 - Committee Rulings
+1. Any additional rules must be voted on and win by a 2/3 majority in order to be ratified.
+2. In order to be a voting member, a player must meet *one* of the following requirements:
+    1. Contributing member to the ping-pong-champion-rules git repo.
+    2. Won a championship title at some time in the past.
+    3. A current Vivid Seats employee that has seen the movie [Balls of Fury](http://www.imdb.com/title/tt0424823/)
+
+## Article 2 - Obtaining Championship Status
+1. Championship matches are played best 3 out of 5, 2 serves each to eleven points as per the [ITTF Handbook](http://www.ittf.com/handbook/).
+2. If a player bests their opponent by three consecutive games, they are obligated to offer a 'winner takes all' round. The winner of the 'winner takes all' round becomes the *CHAMPION*.
+3. If during any championship match a player is beaten 11-0, skunk rules go into effect which automatically make the winning player the *CHAMPION*.
+
+## Article 3 - Immunity & Forfeiture
+1. If a player challenges the *CHAMPION*, the *CHAMPION* may do one of three things:
+    1. Accept the challenge and place the title on the line. Backing out of an accepted challenge results in a forfeit.
+    2. Defer the challenge. The current *CHAMPION* may defer a challenge until the next business day. Note that if the *CHAMPION* is out of office (OOO) the next day, the *CHAMPION* retains Championship status. Additionally, it is up to the challenger to re-initiate the challenge at a mutually agreed upon time. If a time cannot be agreed upon or work duties prevent the match from occurring, the *CHAMPION* retains *CHAMPION*ship status.
```

Comment:
I don't agree with, `if the *CHAMPION* is out of office (OOO) the next day, the *CHAMPION* retains Championship status` I think that the champ must play the challenger with in 24 hours, M-F.

Reply:
That would mean that, for example, because JB is WFH today, he loses his champ status? That sounds lame.